lib/rucio/transfertool/fts3.py

```diff
@@ -797,6 +799,9 @@ def __init__(self, external_host, oidc_account=None, oidc_support: bool = False,
         self.default_lifetime = default_lifetime
         self.archive_timeout_override = archive_timeout_override
 
+        plugins = config_get_list("transfers", "plugins", False, "[]")
+        self.plugins = [FTS3MetadataPlugin(plugin.strip(" ")) for plugin in plugins]
```

Comment:
I wonder if we would ever have the case that this would be multiple different plugins.

Reply:
`fts3tape_plugins` would probably be better for future proofing (and legibility)

Reply:
What I meant here is: Would it ever happen that a community loads multiple plugins? I was operating under the assumption it would be 0 or 1.